clients/cobol-lsp-vscode-extension/src/__tests__/JavaCheckTest.ts

```diff
@@ -27,8 +28,73 @@ describe("Check Java version", () => {
         expect(JavaCheck.isJavaVersionSupported('openjdk version "12.0.1" 2018-10-16')).toBeTruthy();
         expect(JavaCheck.isJavaVersionSupported("java 11 2018-09-25")).toBeTruthy();
         expect(JavaCheck.isJavaVersionSupported("java 12 2019-03-19")).toBeTruthy();
+        expect(JavaCheck.isJavaVersionSupported("java 13 2019-09-17")).toBeTruthy();
+        expect(JavaCheck.isJavaVersionSupported("java 14 2020-03-17")).toBeTruthy();
     });
-    it("checks if Java version is not supported", async () => {
+
+    it("when version is not supported", async () => {
         expect(JavaCheck.isJavaVersionSupported('java version "1.7.0_131"')).toBeFalsy();
+        expect(JavaCheck.isJavaVersionSupported('java version "1.5.0_22"')).toBeFalsy();
+    });
+});
+
+describe("Checks Java installation", () => {
+
+    let javaCheck: JavaCheck;
+    let map: any;
```

Comment:
It can be const

Reply:
We change it according to test-case, so can't be const

Reply:
I don't see where you change the reference to the map object in code.

Reply:
in each test map needs to be clean-up